client/app/components/service-status/service-status-action.component.js

```diff
@@ -0,0 +1,12 @@
+import controller from './service-status-action.controller';
+import template from './service-status-action.html';
+
+angular.module('directives').component('serviceStatusAction', {
```

Comment:
It could be attach to the `App` module instead.

```suggestion
angular.module('App').component('serviceStatusAction', {
```